### PR TITLE
tdrop: unstable-2020-05-14 -> 0.4.0

### PR DESCRIPTION
**Pull request by wedens**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/tdrop/default.nix [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
   pname = "tdrop";
-  version = "unstable-2020-05-14";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
     owner = "noctuid";
     repo = "tdrop";
-    rev = "a9f2862515e5c190ac61d394e7fe7e1039871b89";
-    sha256 = "1zxhihgba33k8byjsracsyhby9qpdngbly6c8hpz3pbsyag5liwc";
+    rev = "bae687d38584824a61e34f12be0a298d74bcb411";
```
Comment (prusnak) [inline pkgs/applications/misc/tdrop/default.nix:12]:
```suggestion
    rev = version;
```

**Review by prusnak — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/tdrop/default.nix (hunk 1 above)
Comment (wedens) [reply pkgs/applications/misc/tdrop/default.nix]: Makes sense. Thanks.

**Comment by prusnak** [comment]
@GrahamcOfBorg build tdrop

**Comment by SuperSandro2000** [comment]
ping @wedens

**Pull request merged**